yunohost_project_organization.md

```diff
@@ -1,41 +1,41 @@
+Installing and administrating a server should be accessible to the largest number of people, without prejudice to the quality and reliability of the software. 
 
 ### Values
 
-#### A free and community-based software
+#### Libre and community-based software
 
-YunoHost is a software under free licence, fully community-based and based on applications which are themselves community-based and often free (roundcube, baikal, etc.)
+YunoHost is a software under copyleft libre licence, fully community-based and based on applications which are themselves community-based, and often gratis (Roundcube, Baïkal, etc.)
 
 #### That everyone can appropriate
 
-Historically, the project has been very close to initiatives which aim at creating a neutral and decentralised Internet. This proximity especially with the [FFDN](https://www.ffdn.org/), has materialised by having some contributors to create the InternetCu.be whose mission is to facilitate self-hosting by providing a complete solution including a service (via a VPN) and a device. This militant aspect does not inhibit commercial software initiatives hereby companies could propose support or hosting.
+Historically, the project has been very close to initiatives aiming to creating a neutral and decentralised Internet. This proximity especially with the [FFDN](https://www.ffdn.org/) materialized when some contributors created the InternetCu.be, whose mission is to facilitate self-hosting by providing a complete solution including a service (via a VPN) and a device. This militant aspect does not inhibit commercial software initiatives, hereby companies could propose support or hosting.
 
 ## YunoHost organisation
 
 ### A theme-based, open structure
 
-The objective of the YunoHost organisation is to allow the largest number of people to contribute to software improvement, whether from a technical point of view (development, application packaging) or otherwise (communication, end-user assistance, documentation, etc.). Inspired by the projects which were reviewed at a recent event (Kodi, Debian, Django, Fedora, Wikipedia, etc.) and by ideas stemming from YunoHost contributors (Jérôme, Bram, opi, scith, ju), a decision was made to organise work by special interest groups, federated thanks to a council to key contributors.
+The objective of the organisation is to allow the greates number of people to contribute to software improvement, whether from a technical point of view (development, application packaging) or otherwise (communication, end-user assistance, documentation, etc.). Inspired by the projects reviewed at a recent event (Kodi, Debian, Django, Fedora, Wikipedia, etc.) and by ideas stemming from YunoHost contributors (Jérôme, Bram, opi, scith, ju), a decision was made to organise work by special interest groups, federated thanks to a council of key contributors.
```

Comment:
```suggestion
The objective of the organisation is to allow the greatest number of people to contribute to the project, whether from a technical point of view (development, application packaging) or otherwise (communication, end-user assistance, documentation, etc.). Inspired by the projects reviewed before the creation of this document (Kodi, Debian, Django, Fedora, Wikipedia, etc.) and by ideas stemming from YunoHost contributors (Jérôme, Bram, opi, scith, ju), a decision was made to organise work by work groups, federated thanks to a council of key contributors.
```

Reply:
Filename is organization, content is all organise/organization